05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+Note this is a consistent but biased estimator. Often the population size is not known so this is a method to estimate a mean without knowing the population size. The estimated standard error of the mean is:
+
+$$se(\bar{y})=\frac{1}{\hat{N}_{pop} } \sqrt{\frac{N^2 (1-\frac{n}{N})}{n}\frac{1}{n-1} \sum_{i=1}^n (M_i\bar{y}_i -\hat{t}/N)^2  + \frac{N}{n}  \sum_{i=1}^n \frac{M_i^2}{m_i}\left(1-\frac{m_i}{M_i}\right)s^2_i }$$
+where $\hat{N}_{pop}$ is the estimated population size, $\hat{t}$ is the estimated total, and $s_i^2$ is the sample variance of cluster $i$.
+
+For estimates of proportions, the estimated proportion is:
+
+$$\hat{p}=\frac{\sum_{i=1}^n M_i \hat{p}_i}{ \sum_{i=1}^n M_i}$$
+
+and the associated standard error estimate is:
+
+$$se(\hat{p})=\frac{1}{\hat{N}_{pop} } \sqrt{\frac{N^2 (1-\frac{n}{N})}{n}\frac{1}{n-1} \sum_{i=1}^n (M_i\hat{p}_i -\hat{t}/N)^2  + \frac{N}{n}  \sum_{i=1}^n \frac{M_i^2}{m_i}\left(1-\frac{m_i}{M_i}\right)s^2_i }$$
+
+where $s^2_i$ is defined as:
+
+$$s^2_i = \frac{m_hp_h(1-p_h)}{m_h-1}$$.
+
+#### The syntax {-} 
+
+To specify a two-stage clustered design without replacement,  use the following syntax:
+
+
+```{r, eval=FALSE}
+des_clus2 <- dat %>%
+  as_survey_design(weights = wtvar, ids = c(PSU, SSU), fpc = c(N, M))
+```
+
+where `SSU` and `PSU` are the variables indicating the PSU and SSU identifiers and `N` and `M` are the variables indicating the population sizes - the number of clusters and then the number of units within each cluster. Note that `N` will be the same for all records (within a strata) and `M` will be the same for all records within the same cluster.
+
+If clusters were sampled with replacement or from a very large population, a fpc is not necessary. Additionally, only the first stage of selection is necessary. The syntax below will yield the same estimates in the end:
+
+```{r}
+#| label: clus2wr
+#| eval: FALSE
+des_clus2wra <- dat %>%
+  as_survey_design(weights = wtvar, ids = c(PSU, SSU))
+
+des_clus2wrb <- dat %>%
+  as_survey_design(weights = wtvar, ids = PSU)
+
+```
+
+#### Example {-} 
+
+The `survey` package includes a two-stage cluster sample data, `apiclus2`, in which school districts were sampled and then a random sample of 5 schools was selected within each district. For districts with fewer than 5 schools, all schools were sampled. School districts have identifiers `dnum` and schools have identifiers of `snum`. The variable `fpc1` indicates how many districts there were in California and `fpc2` indicates how many schools were in a given district with at least 100 students.  The data has a row for each school. In the data printed below, there are 757 school districts as indicated by `fpc1` and there are 9 schools in district 731, one school in district 742, 2 schools in district 768, 6 schools in district 781, and 1 school in district 795 as indicated by `fpc2`.
+
+```{r}
+#| label: api2clus_dis
+apiclus2_slim <- 
```

Comment:
As above, why are you creating this?  May want to provide a little input as to why you're doing this.